node/src/BaseClient.ts

```diff
@@ -1315,12 +1315,12 @@ export class BaseClient {
      *
      * See https://valkey.io/commands/bitfield_ro/ for more details.
      *
+     * Since Valkey version 6.0.0.
```

Comment:
```suggestion
     * @since Valkey version 6.0.0.
```
Please check what we need to configure for eslint and tsdoc to get @since tag accepted. We will apply this for `@see` too.